rllib/examples/grpc_cartpole/cartpole_pb2.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```

Comment:
you can set up a bazel rule so this file will be generated automatically from your .proto file when the test rule is run, so you don't have to include this strange-looking file.
this is of course minor, just saying.